README.md

```diff
@@ -49,7 +49,7 @@ You can see a (customized) instance of Strapping in action by visiting http://oV
 2. Clone the repository:
 
    ```
-   git clone https://github.com/OSAS/strapping-mediawiki strapping
+   git clone https://github.com/axxie/strapping-mediawiki strapping
```

Comment:
Are you sure you want https://github.com/OSAS/strapping-mediawiki to link to axxie?

Thank you for working on strapping-mediawiki!